src/main/kotlin/datamaintain/Filter.kt

```diff
@@ -0,0 +1,7 @@
+package datamaintain
+
+class Filter(private val config: Config) {
+    fun <T : Script> filter(scripts: List<T>): List<T> {
+        return scripts.filterNot {script -> config isScriptBlacklisted script }
```

Comment:
I find infix overkill in this case. I would prefer `config.isScriptBlacklisted(script)` (it is subjective consideration)

Reply:
I moved matching logic into tag